api.md

```diff
@@ -160,6 +160,7 @@ return <primitive object={mesh} position={[0, 0, 0]} />
 #### Using 3rd-party objects declaratively
 
 The `extend` function extends three-fibers catalogue of JSX elements.
+Much like any primitive, components added to the catalog via `extend` are expected to use camel casing.
```

Comment:
Thanks for following up on this. I had small changes to wording to reduce redundancy.
```suggestion
The `extend` function extends three-fiber's catalogue of JSX elements. Components added this way can then be referenced in the scene-graph using camel casing similar to other primitives.
```